Add files and reorganize

birdday opened the pull request:
Primarily restructured the repository to make the workflow more clear and intuitive. Several extraneous/outdated files were removed. Conversely, many of the isc helper files I've written and used locally were added to the repo. Changes were made to the files used to create/submit RASPA simulations to reflect the changes I had made locally. Finally, some data paths were changed to either reflect the reorganized directories, or simply to test code after reorganizing. 

Plans for the future to eliminate all hard-coded data paths with specific config files. Not, implemented here.

birdday committed:
Reoganized Files.

birdday committed:
Changes to '.gitignore'.

Removed all unnecessary file types and directories to prevent not tracking key files in the future.

birdday committed:
Removed extraneous and outdated files.

birdday committed:
Changes to 'README.md'.

Still needs updating. Squash with later changes.

paulboone commented on .gitignore:
```diff
@@ -1,21 +1,8 @@
+*.csv
+
+# Prevents tracking files needed for python modules.
+__init__.py
```
__init__.py is a real code file that should never be ignored. You need that.

paulboone commented on Simulation_Files/settings/process_config.extra_properties.yaml:
```diff
@@ -0,0 +1,424 @@
+# edit this file and save as process_config.yaml when running locally
+# contains how many gas mixtures are added when interpolating, the range that is used for
```
put sample configuration files in a "examples" / "templates" / "settings" / "samples"(?) in the root directory. Or it can even go in the docs/examples ?

paulboone commented on Array_Analysis/Genetic_Algorithm/GeneticAlgorithm_Array.py:
```diff
@@ -0,0 +1,182 @@
+"""
```
no caps. genetic_algorithm with an _

paulboone commented on Array_Analysis/Genetic_Algorithm/GeneticAlgorithm_BrianEdits.py:
```diff
@@ -0,0 +1,594 @@
+"""
```
this replaces other one later; can be squashed with other

paulboone commented on Misc_Helper_Files/csv_check.py:
```diff
@@ -0,0 +1,116 @@
+# Written by Brian Day
```
better filename?

paulboone commented on Array_Analysis/execute_genetic_algorithm_analysis.py:
```diff
@@ -19,15 +19,16 @@
 # 5 and 10 bar
-sim_data = '/Users/brian_day/Desktop/WilmerLab_Research/Research Projects/CO2 Sensing/CO2_RASPA_Results/CO2_5bar/CO2_Sensing_Results_New/ALL_MOFS.csv'
-exp_data = '/Users/brian_day/Desktop/WilmerLab_Research/Research Projects/CO2 Sensing/CO2_RASPA_Results/CO2_5bar/CO2_Sensing_Results_New/exp_data_175.csv'
+sim_data = '/Users/brian_day/Desktop/WilmerLab_Research/Research Projects/CO2 Sensing/CO2_RASPA_Results/CO2_10bar/CO2_Sensing_Results_New/ALL_MOFS.csv'
```
use git stash, don't commit

paulboone commented on Array_Analysis/config_files/analysis_config.yaml:
```diff
@@ -4,6 +4,8 @@
 # densities of mofs, experimental masses for each mof, the mofs included in the analysis, the gases
 # considered, and the mofs in the end array
 
+# Adding a comment to check if changes have been tracked.
```
git reset HEAD^ (i think) and don't push

paulboone commented on Array_Analysis/execute_brute_force_analysis.py:
```diff
@@ -30,15 +30,15 @@
-sim_data = 'ALL_MOFS.csv'
-exp_data = 'exp_data_175.csv'
+sim_data = '/Users/brian_day/Desktop/WilmerLab_Research/Research_Projects/Gas_Sensing/CO2_RASPA_Results/CO2_10bar/CO2_Sensing_Results_New/ALL_MOFS.csv'
+exp_data = '/Users/brian_day/Desktop/WilmerLab_Research/Research_Projects/Gas_Sensing/CO2_RASPA_Results/CO2_10bar/CO2_Sensing_Results_New/exp_data_175.csv'
```
obviously we should do proper command line scripts using setup.py

paulboone commented on .gitignore:
```diff
@@ -6,3 +6,6 @@
 __pycache__
+
+# Prevents tracking of yaml config files.
+*.yaml
```
you want to leave yaml files?

paulboone replied:
ok for now, we can fix this later

paulboone reviewed (changes requested):
1) ~~add top-level module~~
2) ~~rename submodules~~
3) ~~add back in the __init__.py's~~
4) move python run as command line scripts to a ./cli => we will do this later when we do the commands
5) ~~eliminate crappy commits~~
2) ~~move settings / requirements to base-level~~

birdday committed:
Added Genetic Algorithm files.

birdday committed:
Added a series of misc helper files.

birdday committed:
Removed mof_array directory.

birdday committed:
Removed duplicate and unused files.

birdday committed:
Removed yaml files from settings directory. Now in config files, but untracked by git.

birdday committed:
Removed unnecessary __init__.py from settings directory.

birdday committed:
Added and modified files for submitting RASPA simulations.

birdday committed:
Added top level directory. Renamed sub-packages.

Yaml file showing as deleted rather than renamed since it is no longer tracked by git ignore (I think).

birdday committed:
Added __init__.py files, moved settings and requirements directories.
